### ISSUE TITLE
Automatical discovery of batch submission system

### ISSUE THREAD

**Issue by grzanka** [DataMedSci/mcpartools#41]
User can be unaware which batch submission system is installed. Add a mechanism which automatically discovers that and generates scripts accordingly.

**Comment by ant6** [comment]
@grzanka is this a good place to 'check' what bacth system do we have and then return Slurm, Torque etc?
https://github.com/DataMedSci/mcpartools/blob/master/mcpartools/scheduler/common.py#L13

**Comment by ant6** [comment]
I would also e.g. execute `qstat` to check if it returns zero or not, and then decide. Like:
```
[xxx@login01 ~]$ srun -V
slurm 16.05.4
[xxx@login01 ~]$ qstat -h
-bash: qstat: command not found
```
Now we probably know that we can use Slurm. I used qstat, not qsub, because the latter one is slow. No idea why.

**Comment by grzanka** [comment]
Yes, https://github.com/DataMedSci/mcpartools/blob/master/mcpartools/scheduler/common.py#L13 is a good place. 

Try `man qsub` first. `qsub -V` is not priting version, its doing sth different.

**Comment by grzanka** [comment]
On the other hand - I like the idea of executing `srun` / `qsub` to discover queuing system. You can use more than one check - for example check if `--help` and `--version` works (if present).

**Comment by ant6** [comment]
Closed i #48
